src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -80,6 +80,8 @@ class DefaultStorageKeyAllocator
     static const char * OTACurrentProvider() { return "o/cp"; }
     static const char * OTAUpdateToken() { return "o/ut"; }
 
+    static const char * DNSExtendedDiscoveryTimeout() { return "dns/edt"; }
```

Comment:
I think the intent here was to have "global" (non-fabric-specific) keys start with "g/" while fabric-specific ones start with "f/fabric-index/"....  It looks like some keys have snuck in not following those conventions, but we should fix that, and probably make this one "g/dns/edt" or so.

Reply:
Ok, I wasn't aware of that. Fixed.